evidence/mocks/block_store.go

```diff
@@ -4,6 +4,7 @@ package mocks
 
 import (
 	mock "github.com/stretchr/testify/mock"
+
```

Comment:
must have been vs when I tried to format it. Is this a problem?

Reply:
It's just noisy, and it makes cherry-picking harder. For example, if we cherry-pick any changes made to `master` after this change is merged, without cherry-picking this change, then we're going to get spurious merge conflicts for any changes that were made in the vicisinity of these. This problem compounds with the number of "noisy" PRs made, creating lots of manual work for us to fix merge conflicts. PRs should generally not change stuff that's not intended to be changed, and in particular shouldn't make broad auto-reformatting of source code like this.

If we actually want these changes, I suggest submitting a separate PR that formats all of the code like this. Otherwise, consider changing your editor settings so they don't make these changes.

Reply:
I have removed them